docs/usage/configuration-options.md

```diff
@@ -462,8 +487,10 @@ Examples of what having a Dependency Dashboard will allow you to do:
 - Override any rate limiting (e.g. concurrent PRs) or scheduling to force Renovate to create a PR that would otherwise be suppressed
 - Recreate an unmerged PR (e.g. for a major update that you postponed by closing the original PR)
 
-Note: Enabling the Dependency Dashboard does not itself change any of the "control flow" of Renovate, e.g. it will otherwise still create and manage PRs exactly as it always has, including scheduling and rate limiting.
-The Dependency Dashboard therefore provides visibility as well as additional control.
+<!-- prettier-ignore -->
+!!! note
+    Enabling the Dependency Dashboard does not itself change any of the "control flow" of Renovate, e.g. it will otherwise still create and manage PRs exactly as it always has, including scheduling and rate limiting.
+    The Dependency Dashboard therefore provides visibility as well as additional control.
```

Comment:
```suggestion
<!-- prettier-ignore -->
!!! tip
    Enabling the Dependency Dashboard does not itself change any of the "control flow" of Renovate, e.g. it will otherwise still create and manage PRs exactly as it always has, including scheduling and rate limiting.
    The Dependency Dashboard therefore provides visibility as well as additional control.
```

This might be better as a `tip` style admonition as well.